src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/BitFlagsGenerator.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if MERGE_DCJS
```

Comment:
Rather than surrounding whole files with a big ```#if```, should the files just be conditionally included in the .csproj?